harismah/Dwella
# [PR FIX] Adding Supabase Storage to profile images

Images for profile pictures are now stored in supabase storage so everyone can view the image without downloading. For now, only avatar images are stored (no listings) since there is no create listings page yet for users to upload listings images

closes [#191 and #154]

To test:
- Login (check if header contains profile image)
![image](https://github.com/user-attachments/assets/0760a9a5-5968-4ff0-835e-047f45b61df0)
- Go to profile (check if image shows up) 
![image](https://github.com/user-attachments/assets/95eede10-fdf8-4317-bd46-d2ceb6353ec4)
- Click on Edit Profile (check if current avatar image shows up)
![image](https://github.com/user-attachments/assets/64edbe6c-0881-45d7-b58f-a58a1a1910d7)
- Upload a new image (doesn't have to be in project folder) and submit
![image](https://github.com/user-attachments/assets/64bfac2e-4e7f-4e57-8d65-50fb674d0e21)
- Check if new image shows for your profile and header ![image](https://github.com/user-attachments/assets/766ff98d-6e07-4dea-ab8b-6807053aaee5)
- Also check if avatar image shows up correctly for any of the user's posted listings 
![image](https://github.com/user-attachments/assets/b521a3df-32eb-4f9d-a2ea-af2b4c083538)
